src/core/display/Container.js

```diff
@@ -606,17 +606,23 @@ Container.prototype.renderCanvas = function (renderer)
 
 /**
  * Destroys the container
- * @param [destroyChildren=false] {boolean} if set to true, all the children will have their destroy method called as well
+ * @param [opts] {object} Options parameter
+ * @param [opts.children=false] {boolean} if set to true, all the children will have their destroy
+ *      method called as well. 'opts' will be passed on to those calls.
+ *
  */
-Container.prototype.destroy = function (destroyChildren)
+Container.prototype.destroy = function (opts)
 {
+    opts = opts || {};
```

Comment:
Looks like this is the last outstanding question to this PR. Shall I add support for the old signature with a deprecation warning?